.github/workflows/core_build_publish.yml

```diff
@@ -132,16 +133,14 @@ jobs:
         uses: actions/setup-dotnet@v3
         with:
           dotnet-version: ${{ env.DOTNET_VERSION }}
-      - name: Install trx2junit
-        run: dotnet tool install --global trx2junit
       - name: Restore
-        run: dotnet restore
-      - name: Test
-        run: dotnet msbuild /t:RunTestsWithCoverage
-        working-directory: test
+        run: |
+          dotnet restore
+          dotnet tool restore
+      - name: Unit tests
+        run: dotnet test -- --explicit on --filter-not-trait 'integration=true'
```

Comment:
Do we have explicit tests anywhere or is it some kind of future proofing?

Reply:
All the integration and "external service"-integration tests are explicit.